eng/common/scripts/Verify-Links.ps1

```diff
@@ -65,11 +67,11 @@ function ResolveUri ([System.Uri]$referralUri, [string]$link)
   }
 
   $linkUri = [System.Uri]$link;
-  if($resolveRelativeLinks){
+  if(-Not $checkLinkGuidance) {
```

Comment:
It is probably worth adding a comment here on why resolving relative links is gated by the checkLinkGuidance variable.